cddl/ccf-cose-endorsement-service-identity.cddl

```diff
@@ -0,0 +1,28 @@
+ccf-cose-endorsement-tagged = #6.18(ccf-cose-endorsement)
+
+ccf-cose-endorsement = [
+  phdr : bstr .cbor protected_headers,                     ; bstr-wrapped protected headers
+  uhdr : unprotected_headers,                              ; no unprotected headers included
```

Comment:
I don't mind which, but I suggest we pick a convention between snake_case and kebab-case, rather than have a mix.